tests/unit/algorithms/classification/test_helper.py

```diff
@@ -0,0 +1,160 @@
+                label_schema.add_group(single_label_group)
+        else:
+            main_group = LabelGroup(
+                name="labels",
+                labels=not_empty_labels,
+                group_type=LabelGroupType.EXCLUSIVE,
+            )
+            label_schema.add_group(main_group)
+        return label_schema
+
+def generate_cls_dataset(hierarchical=False, number_of_images=10):
+        resolution = (224, 224)
+        if hierarchical:
+            colors = [(0, 255, 0), (0, 0, 255), (255, 0, 0), (0, 0, 0), (230, 230, 250)]
+            cls_names = ["b", "g", "r", "w", "p"]
+            texts = ["Blue", "Green", "Red", "White", "Purple"]
+        else:
+            colors = [(0, 255, 0), (0, 0, 255)]
+            cls_names = ["b", "g"]
+            texts = ["Blue", "Green"]
+        env_labels = [
+            LabelEntity(name=name, domain=Domain.CLASSIFICATION, is_empty=False, id=ID(i))
+            for i, name in enumerate(cls_names)
+        ]
+
+        items = []
+
+        for _ in range(0, number_of_images):
+            for j, lbl in enumerate(env_labels):
+                class_img = np.zeros((*resolution, 3), dtype=np.uint8)
+                class_img[:] = colors[j]
+                class_img = cv.putText(
+                    class_img,
+                    texts[j],
+                    (50, 50),
+                    cv.FONT_HERSHEY_SIMPLEX,
+                    0.8 + j * 0.2,
+                    colors[j - 1],
+                    2,
+                    cv.LINE_AA,
+                )
+
+                image = Image(data=class_img)
+                labels = [ScoredLabel(label=lbl, probability=1.0)]
+                shapes = [Annotation(Rectangle.generate_full_box(), labels)]
+                annotation_scene = AnnotationSceneEntity(kind=AnnotationSceneKind.ANNOTATION, annotations=shapes)
+                items.append(DatasetItemEntity(media=image, annotation_scene=annotation_scene))
+
+        rng = random.Random()
+        rng.seed(100)
+        rng.shuffle(items)
+        for i, _ in enumerate(items):
+            subset_region = i / number_of_images
+            if subset_region >= 0.9:
+                subset = Subset.TESTING
+            elif subset_region >= 0.6:
+                subset = Subset.VALIDATION
+            else:
+                subset = Subset.TRAINING
+            items[i].subset = subset
+
+        dataset = DatasetEntity(items)
+        return dataset
+
+def init_environment(params, model_template, multilabel=False, hierarchical=False):
+        dataset = generate_cls_dataset(hierarchical)
+        labels_schema = generate_label_schema(
+            dataset.get_labels(), multilabel=multilabel, hierarchical=hierarchical
+        )
+        environment = TaskEnvironment(
+            model=None,
+            hyper_parameters=params,
+            label_schema=labels_schema,
+            model_template=model_template,
+        )
+        return environment, dataset, labels_schema
```

Comment:
Label schema can be retrieved from the TankEnvironment object, so there is no need to return it here.

Reply:
oh, okay, I will fix it tomorrow